src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -410,4 +410,19 @@
 
 	<bean id="cuApiJsonWebRequestReader" class="edu.cornell.kfs.sys.web.service.impl.CuApiJsonWebRequestReaderImpl" />
 
+    <!--
+        Override of daily and weekly email reminder batch steps to backport changes from the 2021-04-01 patch.
+        These two bean overrides should be removed when we upgrade to the 2021-04-01 patch or later.
```

Comment:
Should the FINP number be added as well? Thank you.

Reply:
Will update comment.